src/hotspot/share/interpreter/bytecode.cpp

```diff
@@ -157,15 +157,6 @@ Method* Bytecode_invoke::static_target(TRAPS) {
   return LinkResolver::resolve_method_statically(bc, constants, index(), THREAD);
 }
 
-Handle Bytecode_invoke::appendix(TRAPS) {
```

Comment:
You might want to mention the removal of this function in
the bug report so that it will show up in a JBS search.